modules/oadp-configuring-noobaa-for-dr.adoc

```diff
@@ -4,15 +4,17 @@
 
 :_content-type: PROCEDURE
 [id="oadp-configuring-noobaa-for-dr_{context}"]
-= Configuring NooBaa for disaster recovery on {rh-storage}
+= Configuring Multicloud Object Gateway for disaster recovery on {rh-storage}
 
-If you use cluster storage for your NooBaa bucket `backupStorageLocation` on {rh-storage}, configure NooBaa as an external object store.
+If you use cluster storage for your Multicloud Object Gateway (MCG) bucket `backupStorageLocation` on {rh-storage}, configure MCG as an external object store.
```

Comment:
@RichardHoch 
Pedantic:  `backupStorageLocation` is used here (not by you) as meaning "backup storage location". I don't think this is correct. `backupStorageLocation` does not mean 'location', it's the name of a spec. I would suggest changing to:
"If you use cluster storage as your MCG bucket's backup storage location` on {rh-storage}, configure MCG ... " 
But up to you.